deploy/charts/kubernetes-cluster-prep/templates/clusterrole.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.authnK8s.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.authnK8s.rbac.clusterRole }}
+  labels:
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ .Release.Name }}
+    app.kubernetes.io/component: conjur-config
+    app.kubernetes.io/instance: {{ .Release.Namespace }}
+    app.kubernetes.io/part-of: "authn-k8s-cluster-config"
```

Comment:
Let's add this:
```
    app.kubernetes.io/managed-by: helm
```
since that's another standard label, might be useful to someone/something.

Reply:
That's already there. Helm must add it.